yt_dlp/extractor/noice.py

```diff
@@ -0,0 +1,115 @@
+    int_or_none,
+    parse_iso8601,
+    traverse_obj,
+)
+
+
+class NoicePodcastIE(InfoExtractor):
+    _VALID_URL = r'https?://open\.noice\.id/content/(?P<id>[a-fA-F0-9-]+)'
+    _TESTS = [{
+        'url': 'https://open.noice.id/content/7694bb04-ff0f-40fa-a60b-5b39f29584b2',
+        'info_dict': {
+            'id': '7694bb04-ff0f-40fa-a60b-5b39f29584b2',
+            'ext': 'mp3',
+            'season': 'Season 1',
+            'description': 'md5:58d1274e6857b6fbbecf47075885380d',
+            'release_date': '20221115',
+            'timestamp': 1668496642,
+            'season_number': 1,
+            'upload_date': '20221115',
+            'release_timestamp': 1668496642,
+            'title': 'Eps 1. Belajar dari Wishnutama: Kreatif Bukan Followers! (bersama Wishnutama)',
+            'modified_date': '20221121',
+            'categories': ['Bisnis dan Keuangan'],
+            'duration': 3567,
+            'modified_timestamp': 1669030647,
+            'thumbnail': 'https://images.noiceid.cc/catalog/content-1668496302560',
+            'channel_id': '9dab1024-5b92-4265-ae1c-63da87359832',
+            'like_count': int,
+            'channel': 'Noice Space Talks',
+            'comment_count': int,
+            'dislike_count': int,
+            'channel_follower_count': int,
+        }
+    }, {
+        'url': 'https://open.noice.id/content/222134e4-99f2-456f-b8a2-b8be404bf063',
+        'info_dict': {
+            'id': '222134e4-99f2-456f-b8a2-b8be404bf063',
+            'ext': 'mp3',
+            'release_timestamp': 1653488220,
+            'description': 'md5:35074f6190cef52b05dd133bb2ef460e',
+            'upload_date': '20220525',
+            'timestamp': 1653460637,
+            'release_date': '20220525',
+            'thumbnail': 'https://images.noiceid.cc/catalog/content-1653460337625',
+            'title': 'Eps 1: Dijodohin Sama Anak Pak RT',
+            'modified_timestamp': 1669030647,
+            'season_number': 1,
+            'modified_date': '20221121',
+            'categories': ['Cerita dan Drama'],
+            'duration': 1830,
+            'season': 'Season 1',
+            'channel_id': '60193f6b-d24d-4b23-913b-ceed5a731e74',
+            'dislike_count': int,
+            'like_count': int,
+            'comment_count': int,
+            'channel': 'Dear Jerome',
+            'channel_follower_count': int,
+        }
+    }]
+
+    def _get_formats_and_subtitles(self, media_url, video_id):
+        formats, subtitles = [], {}
+        if isinstance(media_url, str):
+            media_url = [media_url]
+        for url in media_url:
+            ext = determine_ext(url)
+            if ext == 'm3u8':
+                fmts, subs = self._extract_m3u8_formats_and_subtitles(url, video_id)
+                formats.extend(fmts)
+                self._merge_subtitles(subs, target=subtitles)
+            else:
+                formats.append({
+                    'url': url,
+                    'ext': 'mp3',
```

Comment:
```suggestion
                    'url': url,
                    'ext': 'mp3',
                    'vcodec': 'none',
                    'acodec': 'mp3',
```
The vcodec is especially important so that yt-dlp can tell it's an audio-only format